specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/AppServiceEnvironments.json

```diff
@@ -493,6 +493,148 @@
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get ASE custom DNS suffix configuration": {
+            "$ref": "./examples/GetAseCustomDnsSuffixConfiguration.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "AppServiceEnvironments"
+        ],
+        "summary": "Update Custom Dns Suffix configuration of an App Service Environment",
+        "operationId": "AppServiceEnvironments_UpdateAseCustomDnsSuffixConfiguration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the App Service Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "customDnsSuffixConfiguration",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/CustomDnsSuffixConfiguration"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/CustomDnsSuffixConfiguration"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update ASE custom DNS suffix configuration": {
+            "$ref": "./examples/UpdateAseCustomDnsSuffixConfiguration.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "AppServiceEnvironments"
+        ],
+        "summary": "Delete Custom Dns Suffix configuration of an App Service Environment",
+        "operationId": "AppServiceEnvironments_DeleteAseCustomDnsSuffixConfiguration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the App Service Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](http://example.com/codeflow?start=9&length=9)

As per ARM's guidelines, the service needs to return a 204 if the resource being deleted does not exist